### DISCUSSION TITLE
Multiple condition check for assign

### DISCUSSION THREAD

**Discussion by shivanshmehta3** [instancio/instancio#808] [Ideas]
```kotlin
assign(
    andConditions(
        given(field(class A, "fieldName1")).is("value1"),
        given(field(class B, "fieldName2")).is("value2"), 
        ...
    ).set()...
)
``` 

Is it possible to add multiple `and` conditional check while assigning value? Something like mentioned above.

**Comment by armandino** [comment]
No, unfortunately it's not supported and there's no plan to implement this as it would add a lot of complexity.
